build.gradle

```diff
@@ -597,6 +597,7 @@ jlink {
         uses 'com.airhacks.afterburner.injection.PresenterFactory'
         uses 'org.controlsfx.glyphfont.GlyphFont'
         uses 'com.airhacks.afterburner.views.ResourceLocator'
+        uses javax.xml.bind.JAXBContextFactory
```

Comment:
add `'` around it?

Reply:
Just noticed the syntax errro and also added a changelog